core/libp2p/common_objects/network_message.hpp

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_NETWORK_MESSAGE_HPP
+#define KAGOME_NETWORK_MESSAGE_HPP
+
+namespace libp2p {
+  namespace common {
```

Comment:
same here - dir is named `common_objects` but namespace is `common`. 
pls rename dir to `common`